src/main/features/dashboard/views/macro/claimStatus/admissionByCCJDefendant.njk

```diff
@@ -64,7 +79,7 @@
     {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
       <p>{{ t('They rejected your repayment plan.') }}</p>
     {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
-      <p>{{ t('They accepted your offer to pay {{ partAdmitAmount }}. They rejected your repayment plan.', { partAdmitAmount: claim.response.amount }) }}</p>
+      <p>{{ t('They accepted your offer to pay {{ partAdmitAmount }}. They rejected your repayment plan.', { partAdmitAmount: claim.response.amount | numeral }) }}</p>
```

Comment:
Does this fix [ROC-5904](https://tools.hmcts.net/jira/browse/ROC-5904)?

Reply:
should do! i noticed it when I was adding the status and fixed it everywhere I saw it.